### ISSUE TITLE
Issue with humidity sensor trigger

### ISSUE THREAD

**Issue by huta76** [RavenSystem/esp-homekit-devices#1284]
I am suspecting there is a bug with the triggerring of actions in the humidity sensor switch.  This is my config using Sonoff Basic R2 and DHT22 sensor.

{
  "c": {
    "n": "HAA-EnSuiteExtractor",
    "l": 13,
    "i": 0,
    "b": [
      {
        "g": 0,
        "t": 5
      }
    ]
  },
  "a": [
    {
      "t": 24,
      "g": 3,
      "y1": [
        {
          "0": {
            "m": [
              {
                "g": 2,
                "v": 1
              }
            ]
          },
          "v": 95,
          "r": 1
        },
        {
          "0": {
            "m": [
              {
                "g": 2,
                "v": 0
              }
            ]
          },
          "v": 80
        }
      ]
    },
    {
      "0": {
        "r": [
          {
            "g": 12
          }
        ]
      },
      "1": {
        "r": [
          {
            "g": 12,
            "v": 1
          }
        ]
      },
      "b": [],
      "s": 0
    },
    {
      "1": {
        "s": [
          {
            "a": 1
          }
        ]
      },
      "b": [],
      "s": 0
    },
    {
      "1": {
        "s": [
          {
            "a": 2
          }
        ]
      },
      "b": [],
      "s": 0
    }
  ]
}

Basically the expected behaviour is to turn on switch (connected to extractor fan) when humidity is 90% and turn off when 80% is reached.   However, the switch is being turned off as soon as humidity reached 89%. 

Anyone has come across this please?

**Comment by huta76** [comment]
I have managed to work around this by creating a "do nothing" action for the humidity % which is +1 more than the required power off value as per below:

{
    "c": {
        "n": "HAA-ShowerExtractor",
        "l": 13,
        "i": 0,
        "b": [
            {
                "g": 0,
                "t": 5
            }
        ]
    },
    "a": [
        {
            "t": 24,
            "g": 3,
            "y1": [
                {
                    "0": {
                        "m": [
                            {
                                "g": 2,
                                "v": 1
                            }
                        ]
                    },
                    "v": 95,
                    "r": 1
                },
                {
                    "0": {
                        "r": []
                    },
                    "v": 81,
                    "r": 1
                },
                {
                    "0": {
                        "m": [
                            {
                                "g": 2,
                                "v": 0
                            }
                        ]
                    },
                    "v": 80
                }
            ]
        },
        {
            "0": {
                "r": [
                    {
                        "g": 12
                    }
                ]
            },
            "1": {
                "r": [
                    {
                        "g": 12,
                        "v": 1
                    }
                ]
            },
            "b": [],
            "s": 0
        },
        {
            "1": {
                "s": [
                    {
                        "a": 1
                    }
                ]
            },
            "b": [],
            "s": 0
        },
        {
            "1": {
                "s": [
                    {
                        "a": 2
                    }
                ]
            },
            "b": [],
            "s": 0
        }
    ]
}

While this works, I think it should not be necessay. @RavenSystem can you confirm please?
